src/api/emit.js

```diff
@@ -15,76 +13,14 @@ function createCustomEvent (name, opts = {}) {
   if (CustomEvent) {
     return new CustomEvent(name, opts);
   }
-
-  var e = document.createEvent('CustomEvent');
+  const e = document.createEvent('CustomEvent');
```

Comment:
maybe we can re-use what we used in named slots for this at a later point in time:
```
    "custom-event-polyfill": "joscha/custom-event-polyfill#de24bd9",
```